charts/clearml-agent/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: clearml-agent
 description: MLOps platform Task running agent
 type: application
-version: "5.1.5"
+version: "5.1.6"
```

Comment:
Since we are adding new parameters in values, pls move this to 5.2.0

Reply:
done